README.rst

```diff
@@ -18,7 +18,7 @@ Typical usage looks like this:
    >>> class my_decorator(Decorum):
    ...    def wraps(self, f):
    ...        print("I'm returning the function! You can keep it!")
-   ...        return f
+   ...        return super(my_decorator, self).wraps(f)
```

Comment:
The way I refactored the code, we have to call super().wraps() in order to take advantage of Decorum's base features: assign `self._wrapped` and assign `__name__` and `__doc__`.